packages/devtools-ui-kit/src/unocss.ts

```diff
@@ -154,6 +154,10 @@ export function unocssPreset(): Preset {
       'n-loading': 'flex h-full w-full justify-center items-center',
       'n-panel-grids': 'n-panel-grids-light dark:n-panel-grids-dark',
       'n-panel-grids-center': 'n-panel-grids flex flex-col h-full gap-2 items-center justify-center',
+
+      // glass
+      'glass-effect': 'backdrop-blur-6 bg-white/80 dark:bg-[#151515]/90',
+      'navbar-glass': 'sticky z-10 top-0 glass-effect',
```

Comment:
Better to always adding `n-` prefix for ui-kit utils

Reply:
okay, thanks. what about the bg-color? does it need to be bg-context?

Reply:
I think it's fine to hard-coded the colors for now, as tinted glass-effects do not make much sense for now.